stats/rpc_constants.textpb

```diff
@@ -0,0 +1,620 @@
+measurement_descriptor {
+  name: "/rpc/client/error_count"
+  description: "Count of client RPCs that received an error (unsuccessful) response"
+  unit: {
+    numerators: SCALAR
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/request_bytes"
+  description: "Request bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/response_bytes"
+  description: "Response bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/roundtrip_latency"
+  description: "RPC roundtrip latency in milliseconds on client RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/server_elapsed_time"
+  description: "Server elapsed time in milliseconds on client RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/uncompressed_request_bytes"
+  description: "Uncompressed Request bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/client/uncompressed_response_bytes"
+  description: "Uncompressed Response bytes of client RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+# Census defined RPC server MeasurementDescriptors.
+measurement_descriptor {
+  name: "/rpc/server/error_count"
+  description: "Count of server RPCs that received an error (unsuccessful) response"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/request_bytes"
+  description: "Request bytes of server RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/response_bytes"
+  description: "Response bytes of server RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/server_elapsed_time"
+  description: "Server elapsed time in milliseconds on server RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/server_latency"
+  description: "Latency in milliseconds on server RPC side"
+  unit: {
+    power10: -3
+    numerators: SECONDS
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/uncompressed_request_bytes"
+  description: "Uncompressed Request bytes of server RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/uncompressed_response_bytes"
+  description: "Uncompressed Response bytes of server RPCs"
+  unit: {
+    numerators: BYTES
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/started_count"
+  description: "Number of server RPCs started"
+  unit: {
+    numerators: SCALAR
+  }
+}
+
+measurement_descriptor {
+  name: "/rpc/server/finished_count"
+  description: "Number of server RPCs finished"
+  unit: {
+    numerators: SCALAR
+  }
+}
+
+# Census defined RPC client ViewDescriptors.
+view_descriptor {
+  name: "rpc client error_count"
```

Comment:
Yep, we are discussing on naming convention on team's doc. I'll update the names once we came to a conclusion.